src/Types/Resources.ts

```diff
@@ -0,0 +1,27 @@
+export interface Resources {
+  label: string; // Name to be displayed to the user
+  name: string; // unique id for reference
```

Comment:
we also need to refer to the trainerId, and enforce the pair of (trainerId, name) to be unique instead